server/test/schema-validation/environment-types.spec.js

```diff
@@ -0,0 +1,73 @@
+'use strict';
+
+let schema = require('../../../client/schema/EnvironmentType.schema.json');
+let Ajv = require('ajv');
+let _ = require('lodash');
+let assert = require('assert');
+let sample = require('./environment-type.sample.json');
```

Comment:
When requiring a JSON file, the extension isn't necessary. 

Although if this was added deliberately for clarity, I actually quite like that too.